### PR TITLE
Experimental: automatically add extensions to new courses

### PR DESCRIPTION
**Pull request by taylortom**
#1875

### CODE REVIEW

File: plugins/content/extension/index.js [hunk 1]
```diff
@@ -461,52 +485,35 @@ function initialize () {
...
-      }
-
-      toggleExtensions(courseId, 'disable', extensions, function(error, result) {
-        if (error) {
+      disableExtensions(req.params.courseid, req.body.extensions, function(error) {
+        if(error) {
+          logger.log('error', error);
           res.statusCode = error instanceof ContentTypeError ? 400 : 500;
-          return res.json({ success: false, message: error.message });
+          return res.status(400).json({ success: false, message: error });
```
Comment (lc-thomasberger) [inline plugins/content/extension/index.js]: isn't statusCode already set above?

**Comment by lc-thomasberger** [comment]
Have tested and works for me. Happy to +1 once the conflict is resolved.

**Comment by canstudios-louisem** [comment]
Can we clarify what this is solving? The linked issue does not match the PR title.

**Comment by taylortom** [comment]
That is the correct issue, but the issue's title is a bit misleading. 

The aim is to be able to flag certain extensions so that they're automatically enabled on new courses.

**Comment by lc-thomasberger** [comment]
I think the feature makes sense. For instance most projects will require Tutor for displaying Feedback.

I also think the opposite would be handy, "disable extensions for publish". For instance, you would use dev-tools in the preview, but it will automatically excluded once you publish a course when the checkbox is enabled.

**Pull request merged**